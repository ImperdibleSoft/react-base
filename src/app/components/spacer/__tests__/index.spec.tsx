import * as React from 'react';
import { shallow } from 'enzyme';

import Spacer from '..';

describe('Spacer', () => {
  it('should render a small spacer without border', () => {
    const component = shallow(<Spacer size="sm" />);
    expect(component).toMatchSnapshot();
  });

  it('should render an extra small spacer without border', () => {
    const component = shallow(<Spacer size="xs" />);
    expect(component).toMatchSnapshot();
  });

  it('should render the default spacer', () => {
    const component = shallow(<Spacer />);
    expect(component).toMatchSnapshot();
  });
});
