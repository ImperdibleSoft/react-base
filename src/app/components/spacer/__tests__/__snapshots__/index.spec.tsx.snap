// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render a small spacer without border 1`] = `
<StyledSpacer
  borderSize={0}
  marginSize={15}
/>
`;

exports[`Spacer should render an extra small spacer without border 1`] = `
<StyledSpacer
  borderSize={0}
  marginSize={10}
/>
`;

exports[`Spacer should render the default spacer 1`] = `
<StyledSpacer
  borderSize={1}
  marginSize={20}
/>
`;
